build-system/tasks/css/index.js

```diff
@@ -15,27 +15,21 @@
  */
 
 const debounce = require('debounce');
-const file = require('gulp-file');
 const fs = require('fs-extra');
-const gulp = require('gulp');
-const {
-  endBuildStep,
-  mkdirSync,
-  toPromise,
-  watchDebounceDelay,
-} = require('../helpers');
+const globby = require('globby');
+const path = require('path');
 const {buildExtensions} = require('../extension-helpers');
+const {endBuildStep, watchDebounceDelay} = require('../helpers');
 const {jsifyCssAsync} = require('./jsify-css');
 const {maybeUpdatePackages} = require('../update-packages');
-const {watch} = require('gulp');
+const {watch} = require('chokidar');
 
 /**
  * Entry point for 'gulp css'
- * @return {!Promise}
```

Comment:
This is true, but from what I've seen, our `eslint` JSDoc enforcer requires that a `@return` annotation be added only for functions that actually contain a `return` statement. Functions that are `async` return an implicit Promise, and I think it would be superfluous to document each of them with `@return {!Promise}`.

Reply:
Noted! If we ever switch to TS though, it won't be as forgiving

Reply:
Interesting point. Will keep this in mind if / when we start enforcing TS-style annotations for our build system codebase.